### PR TITLE
return total number of verifications

### PR DESCRIPTION
**Pull request by arcofdescent**

### CODE REVIEW

File: lib/Experian/IDAuth.pm [hunk 1]
```diff
@@ -387,6 +387,8 @@ sub _get_result_proveid {
     my $kyc_dob = $kyc_summary->findvalue('DateOfBirth/Count') || 0;
     my $cr_total = $credit_reference->findvalue('TotalNumberOfVerifications')
       || 0;
+    $decision->{num_verifications} = $cr_total;
```
Comment (tfoertsch) [inline lib/Experian/IDAuth.pm]: If we have the requirement to provide the field, we need a test case for it. And we need documentation.

**Comment by tfoertsch** [comment]
Can you please overhaul the POD. Look here <https://metacpan.org/pod/Experian::IDAuth>.

1. find the format errors. First thing that pops up in my eyes is the URL that is not a link. Such things.

1. the page looks impressive but it actually contains very little information about the module. Most of the stuff is just blah blah. All the sections *CHANGES FROM 1.X*, *BUGS* and *SUPPORT* are superfluous. Nobody in the company cares about all these bug trackers etc. So, remove that crap.

1. try to look at the POD tuning out your knowledge of the module and try to understand what it does. If you don't understand, tune in your knowledge and improve the documentation. Repeat these steps until the documentation is complete and comprehensive.

**Comment by arcofdescent** [comment]
Yeah, ok. Moving this back to development. But I needed this release as we need the PEP check. Anyway, I will improve the POD.

**Comment by tfoertsch** [comment]
One more thing. I think the module has a UTF8 problem. The way it generates the XML request looks very suspicious.

I won't force you to fix that because the module works for us. But a note in the documentation would be good.

**Pull request merged**